### PR TITLE
Fix to allow disabling the enforcement of sudo-capable logins

### PR DESCRIPTION
**Pull request by iliajie**
Hello Jamie!

This PR addresses a problem where sudo-capable users might not always want to use sudo or su to gain root access in Terminal.

### CODE REVIEW

File: xterm/acl_security.pl [hunk 1]
```diff
@@ -10,11 +10,20 @@ sub acl_security_form
 print &ui_table_row($text{'acl_user'},
 	&ui_opt_textbox("user", $o->{'user'} eq '*' ? undef : $o->{'user'},
 			20, $text{'acl_sameuser'}));
+
+if ($o->{'user'} eq "root" && $remote_user ne $o->{'user'}) {
```
Comment (jcameron) [inline xterm/acl_security.pl:14]: We should just always show all fields in the ACL, not make them conditional on other fields.

Comment (iliajie) [reply xterm/acl_security.pl:14]: Well, this option has nonsense for a regular user, right?

Comment (jcameron) [reply xterm/acl_security.pl:14]: Right, but the `$o->{'user'}` field that controls if this field is visible is also editable on the ACL form. So it the admin changes the `user` field from something else to `root` he won't be able to adjust this option without saving and then re-opening the page.

Comment (iliajie) [reply xterm/acl_security.pl]: Oh, that's right. What about this fix then https://github.com/webmin/webmin/pull/2195/commits/a12f385a5bc9ba1c2c8dc81fe42126546fdaa18c.

Comment (jcameron) [reply xterm/acl_security.pl]: Just keep it simple and always show that field ... no need for any if statement

Comment (iliajie) [reply xterm/acl_security.pl]: Well .. okay!

**Comment by jcameron** [comment]
So just to clarify, by default any user who has logged into Webmin as a sudo-capable root user and opens a terminal will get a shell running as that user rather than root. Unless this option is changed?

**Comment by iliajie** [comment]
> So just to clarify, by default any user who has logged into Webmin as a sudo-capable root user and opens a terminal will get a shell running as that user rather than root. Unless this option is changed?

Yes, exactly like that!

**Pull request merged**